components/UseCaseCard.vue

```diff
@@ -1,19 +1,31 @@
 <template>
-  <div class="flex p-4 shadow-xl">
+  <div class="flex flex-row overflow-auto p-4 shadow-xl">
     <!-- Usecase cards -->
-    <div
-      v-for="card in panel"
-      :key="card.id"
-      class="flex no-wrap text-left"
-    >
-      <NuxtLink :to="card.route">
-        <h4>
-          {{ card.title }}
-        </h4>
-        <div class="m-2 bg-white rounded">
-          <img :src="card.image" class="object-contain w-40 h-30 mx-auto">
-        </div>
-      </NuxtLink>
+    <div class="no-wrap text-left overflow-auto">
+      <h4 class="text-2xl m-8">
+        {{ title }}
+      </h4>
+      <p class="text-left m-8">
+        {{ description }}
+      </p>
+    </div>
+    <div class="container mx-auto w-1/2 m-3">
+      <vueper-slides
```

Comment:
Hi Peter, thanks for the point. I've tried hard to fit the image inside the slider. But till now I didn't find a solution for it. The Vueper Slides does not provide an option to "contain" the image. I tried with the typical css setup to keep the ratio of the image. The `object-fit: contain;` property, which is supposed to work, is not working here. Other options, like fix the height and keep the aspect ratio, are also not working here. I will keep searching if there is any good way to do it. But would you also have time to take a look? Just spent too much time on this. Thanks.

Reply:
The problem is solved by controlling the css of background rather than the image. Thanks @Peter9192 for your help!! The image can be centered by choosing the same height for both the image and the slider. It works well now.